test/lib/ansible_test/_data/completion/docker.txt

```diff
@@ -1,13 +1,13 @@
-default name=quay.io/ansible/default-test-container:2.7.0 python=3.6,2.6,2.7,3.5,3.7,3.8,3.9 seccomp=unconfined context=collection
-default name=quay.io/ansible/ansible-base-test-container:1.6.0 python=3.6,2.6,2.7,3.5,3.7,3.8,3.9 seccomp=unconfined context=ansible-base
+default name=quay.io/ansible/default-test-container:2.9.0 python=3.6,2.6,2.7,3.5,3.7,3.8,3.9 seccomp=unconfined context=collection
+default name=quay.io/ansible/ansible-base-test-container:1.7.0 python=3.6,2.6,2.7,3.5,3.7,3.8,3.9 seccomp=unconfined context=ansible-base
```

Comment:
A changelog entry is needed for these updates.

Also, I'd backport them to stable-2.10. The other chagnes are probably OK to backport as well, but if not, they should be split into a separate PR.

Reply:
Updated changelog entry to call out those two containers. And I was planning to backport everything to 2.10.

Reply:
The default containers also move from Python 3.9 RC1 to GA, so that should be included in the changelog.